src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.vb

```diff
@@ -1280,7 +1280,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
                    specialtype <> SpecialType.System_Enum AndAlso specialtype <> SpecialType.System_MulticastDelegate Then
                     Dim base As TypeSymbol = GetDeclaredBase(Nothing)
 
-                    If base?.SpecialType = SpecialType.None AndAlso base.ContainingAssembly?.IsMissing Then
+                    If base IsNot Nothing AndAlso base.SpecialType = SpecialType.None AndAlso base.ContainingAssembly?.IsMissing Then
```

Comment:
>base IsNot Nothing AndAlso base.SpecialType = SpecialType.None [](start = 23, length = 62)

This is a fix, we are going to dereference ```base``` after AndAlso. Prior to this PR, compiler would incorrectly apply GetValueOrDefault call.